demo/addons/godotopenxr/.export/meta/godot_openxr_meta_editor_export_plugin.gd

```diff
@@ -79,9 +130,28 @@ func _get_export_options(platform) -> Array[Dictionary]:
 		HAND_TRACKING_OPTION,
 		HAND_TRACKING_FREQUENCY_OPTION, 
 		PASSTHROUGH_OPTION,
+		SUPPORT_QUEST_1_OPTION,
+		SUPPORT_QUEST_2_OPTION,
+		SUPPORT_QUEST_3_OPTION,
+		SUPPORT_QUEST_PRO_OPTION,
 	]
 
 
+func _get_supported_devices() -> PackedStringArray:
+	var supported_devices = PackedStringArray()
+
+	if _get_bool_option("meta_xr_features/quest_1_support"):
+		supported_devices.append("quest")
+	if _get_bool_option("meta_xr_features/quest_2_support"):
+		supported_devices.append("quest2")
+	if _get_bool_option("meta_xr_features/quest_3_support"):
+		supported_devices.append("quest3")
+	if _get_bool_option("meta_xr_features/quest_pro_support"):
+		supported_devices.append("questpro")
+
```

Comment:
We should add a check/warning here when both quest 1 and quest 3 are ticked?

Reply:
Let's leave it be as is for now until it's part of the official documentation.

Reply:
That makes sense